ext/mysql2/result.c

```diff
@@ -169,6 +169,160 @@ static VALUE rb_mysql_result_fetch_field(VALUE self, unsigned int idx, int symbo
   return rb_field;
 }
 
+static VALUE rb_mysql_result_fetch_field_type(VALUE self, unsigned int idx) {
+  VALUE rb_field_type;
+  GET_RESULT(self);
+
+  if (wrapper->fieldTypes == Qnil) {
+    wrapper->numberOfFields = mysql_num_fields(wrapper->result);
+    wrapper->fieldTypes = rb_ary_new2(wrapper->numberOfFields);
+  }
+
+  unsigned long mb_per_char = 3;
```

Comment:
This is fixed for now. Could be doing something like https://github.com/mysql/mysql-connector-j/blob/16a712ddb3f826a1933ab42b0039f7fb9eebc6ec/src/main/core-api/java/com/mysql/cj/CharsetMapping.java for correctness.

Reply:
Please make this a constant, since there's no other values for mb_per_char at this time.